src/aap_eda/tasks/orchestrator.py

```diff
@@ -26,65 +29,67 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def _manage_activation_job_id(activation_id: int) -> str:
+def _manage_activation_job_id(process_parent_type: str, id: int) -> str:
     """Return the unique job id for the activation manager task."""
-    return f"activation-{activation_id}"
+    return f"{process_parent_type}-{id}"
```

Comment:
@bzwei Will this be tricky when we are in the process of migration the names are going to change? Or it won't matter because when we stop all our services and restart we won't have RQ jobs with the old job_id. Is this job_id stored in the DB?

Reply:
job_id is not stored in our DB. I think a restart will clear everything